app/js/views/main.js

```diff
@@ -151,14 +204,75 @@ export default class MainView extends View {
     return mainViewTemplate;
   }
 
-  render() {
-    super();
-
-    this.el.appendChild(this.actionMenuView.el);
+  _addChildViews() {
```

Comment:
Why do we need to remove all of these child views from the Customizer? As long as the ```customizer``` element is out of the DOM tree, we don't gain anything by unlinking the child views as well.